app/src/main/java/com/example/concordia_campus_guide/Models/Campus.java

```diff
@@ -0,0 +1,5 @@
+package com.example.concordia_campus_guide.Models;
+
+public enum Campus {
```

Comment:
maybe we could use typedef, what do you think? by introducing more and more enums, it might slow our app. Look at this: https://android.jlelse.eu/android-performance-avoid-using-enum-on-android-326be0794dc3

Reply:
Oh I'll delete this enum. I added it in the beginning and then decided to not use it. So now it's not being used anywhere